packages/vite/package.json

```diff
@@ -71,12 +71,12 @@
   "homepage": "https://vitejs.dev",
   "funding": "https://github.com/vitejs/vite?sponsor=1",
   "scripts": {
-    "dev": "rimraf dist && pnpm run build-bundle -w",
+    "dev": "rimraf dist && tsx scripts/dev.ts",
```

Comment:
The dev script already deletes `dist` by default, I think we can remove this `rimraf dist` too.